### DISCUSSION TITLE
Large impact tests

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15011] [General]
Gents, did you tried scenario where in the same time 1000 api calls create records in some module?
If not, should we expect some issues?

Thanks

**Comment by TonyM15** [comment]
Interesting

**Comment by mariuszkrzaczkowski** [comment]
The API has not been tested yet, but we did test the CRM workload with 2,000 users actively using CRM without any problems (total number of users was 12,000).
So it's practically the same because the API lacks basic user operations
